src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -284,6 +285,21 @@ public function getTitle()
         return $this->JobTitle;
     }
 
+    /**
+     * Return a string representation of the numeric JobType
+     * @return string
+     */
+    public function getJobTypeString()
+    {
+        $map = [
+            QueuedJob::IMMEDIATE => 'Immediate',
+            QueuedJob::QUEUED => 'Queued',
+            QueuedJob::LARGE => 'Large',
+        ];
```

Comment:
Do you think we should make these translatable as well?

Reply:
Yeah, makes sense.

Reply:
Done.